README.md

```diff
@@ -19,8 +19,10 @@ Playbooks are located in the `playbooks/` directory.
 | Playbook                                 | Inventory                             | Purpose                                                            |
 | ---------------------------------------- | ------------------------------------- | ------------------------------------------------------------------ |
 | `virthost-setup.yml`                     | `./inventory/virthost/`               | Provision a virtual machine host                                   |
-| `kube-install.yml`                       | `./inventory/vms.local.generated`     | Install and configure a k8s cluster                                |
-| `kube-teardown.yml`                      | `./inventory/vms.local.generated`     | Runs `kubeadm reset` on all nodes to tear down k8s                 |
+| `bmhost-setup.yml`                       | `./inventory/bmhost/`                 | Provision a bare metal host and add to group nodes.                |
+| `allost-setup.yml`                       | `./inventory/allhosts/`               | Provision both a virtual machine host and a bare metal host.       |
```

Comment:
`allost-setup.yml` -> `allhost-setup.yml`?

Reply:
This is fixed in PR #293.